### ISSUE TITLE
Multiple bright DNA objects

### ISSUE THREAD

**Issue by tischi** [tischi/spindle3d#11]
The algorithm currently fails if there are multiple bright DNA objects.

Examples:
- https://www.dropbox.com/s/ahoupo47jf84sqt/20201006_R1E309_TubGFP_KATNB1_D5_005-2.tif?dl=0

**Comment by TobiasKletter** [comment]
More examples
https://www.dropbox.com/s/9is3r2vpbchcccl/20200806-164803_HighZoom--W0000--P0001-T0008--0003-1.tif?dl=0
https://www.dropbox.com/s/iffm060pvy3u6ss/20200806-164803_HighZoom--W0000--P0004-T0070--0002-1.tif?dl=0

**Comment by tischi** [comment]
### 3D Screenshots

![image](https://user-images.githubusercontent.com/2157566/96097901-cfab5d00-0ed1-11eb-8616-9dffa05294ba.png)

**Comment by tischi** [comment]
I thought about it for about one hour (I also found a CATS project that we started for this).

But I cannot make up my mind what to do. I find adding machine learning somehow not such a good option because one never knows for which kind of data it will work and also it is quite some work to paint all the ground truth.

I am not sure, maybe we do have an idea how to do this more manually?

I guess somehow combining the tubulin and the DNA signal could be the way to go (before or after applying some filters and before or after applying some thresholds). The aim is just to get a rough idea of where in the image the metaphase plate is. The final segmentation can be done with the code that we have already.


Could you maybe play a bit with this and let me know?

**Comment by TobiasKletter** [comment]
> But I cannot make up my mind what to do. I find adding machine learning somehow not such a good option because one never knows for which kind of data it will work and also it is quite some work to paint all the ground truth.

I remember, when we were playing with CATS and single-decision trees, we found there was one single filter setting that allowed for surprisingly good prediction. I think it was some Hessian filter or something detecting tubeness? I was playing with tubeness in Fiji a bit (plugins->analyze->tubeness), the results aren't too clear-cut unfortunately, but maybe there is something similar or a combination of filters that help...

> I guess somehow combining the tubulin and the DNA signal could be the way to go (before or after applying some filters and before or after applying some thresholds). The aim is just to get a rough idea of where in the image the metaphase plate is. The final segmentation can be done with the code that we have already.

Yes I also feel this is the way to go and I'm going to play around with this in mind

**Comment by tischi** [comment]
According to CATS, you can try this

## DNA channel

FeatureJ:
![image](https://user-images.githubusercontent.com/2157566/96110258-e6f14700-0edf-11eb-8041-51988c3a8d99.png)

## Spindle channel

FeatureJ:
![image](https://user-images.githubusercontent.com/2157566/96110504-3cc5ef00-0ee0-11eb-9e5d-35942cac6614.png)

Maybe also try to do a `[ Process > Math > Log ]` before you do those filters.

And then maybe we just add those two images and find the maximum?

Good luck!

**Comment by tischi** [comment]
... also play with the "Integration Scale" probably a higher value is better!

**Comment by TobiasKletter** [comment]
I will, thanks!

I also found something:

Splitting the channels and using the image calculator:
C1 Minimum C2
![Screenshot 2020-10-15 at 12 20 09](https://user-images.githubusercontent.com/45814595/96111112-0dfc4880-0ee1-11eb-911f-f1ed792559be.png)

This helped to remove the bright DNA signals since the tubulin signal is often pretty low there...

Median-filtering a bit to show the new center of intensity:
![Screenshot 2020-10-15 at 12 20 33](https://user-images.githubusercontent.com/45814595/96111242-3e43e700-0ee1-11eb-9b8a-176a0da11928.png)

I might try this on other busy images...
Do think this could make sense?

**Comment by TobiasKletter** [comment]
![image](https://user-images.githubusercontent.com/45814595/96113322-549f7200-0ee4-11eb-98a6-27fb3220f126.png)
![image](https://user-images.githubusercontent.com/45814595/96113914-253d3500-0ee5-11eb-8661-9d23676c0e5e.png)


So far, this strategy seems to always leave considerable signal in the spindle center where tubulin and DNA overlap, whereas bright nuclei and apoptoc DNA is reduced...

**Comment by tischi** [comment]
I will answer your question later, but just one other idea (based on chatting with @ssgpers):
What about a z-maximum projection and template matching?!

**Comment by tischi** [comment]
I don't get this: `C1 Minimum C2` Could you rephrase? Do you mean `Tubulin minus DNA`?

**Comment by tischi** [comment]
> I think the algorithm here compares voxel brightness between the channels and takes the minimum

Ah, interesting! This however only would work if the images have a similar overall intensity, isn't it?
For example, if you were to reduce the detector gain of one channel during acquisition you would get entirely different results.
Maybe we would have to normalise them somehow before doing this?

We could compute the median and median absolute deviation of each image before and then normalise like this:

`im_norm = ( im - median ) / mad`

`im_norm` should be float type (32 bit) for this to work.

**Comment by TobiasKletter** [comment]
For testing, I processed the images now with the following IJ code. I didn't know how to do it in 3D so I created a median projection of the stack and got the median gray value of the projection. I couldn't find a way to easily measure the MAD so I took the standard deviation instead:

```
inputDir = "/path/to/inputfolder";
outputDir = "/path/to/outputfolder";

files = getFileList(inputDir);

setBatchMode(true);
for (i = 0; i < files.length; i++) 
{
	path = inputDir + "/" + files[ i ];
	print("Opening: " + path);
	open(path);
	run("Duplicate...", "duplicate channels=1");
	rename("Channel_1");
	run("32-bit");
	run("Z Project...", "projection=[Median]");
	List.setMeasurements("limit");
	median_channel_1 = List.getValue("Median");
	stdev_channel_1 = List.getValue("StdDev");
	print("Median of channel 1= " + median_channel_1);
	print("StDev of channel 1 (TODO MAD?!)= " + stdev_channel_1);
	selectWindow("Channel_1");
	run("Subtract...", "value=median_channel_1 stack");
	run("Divide...", "value=stdev_channel_1 stack");
	
	selectWindow(files[i]);
	run("Duplicate...", "duplicate channels=2");
	rename("Channel_2");
	run("32-bit");
	run("Z Project...", "projection=[Median]");
	List.setMeasurements("limit");
	median_channel_2 = List.getValue("Median");
	stdev_channel_2 = List.getValue("StdDev");
	print("Median of channel 2= " + median_channel_2);
	print("StDev of channel 2 (TODO MAD?!)= " + stdev_channel_2);
	selectWindow("Channel_2");
	run("Subtract...", "value=median_channel_2 stack");
	run("Divide...", "value=stdev_channel_2 stack");
	
	imageCalculator("Min create stack", "Channel_1", "Channel_2");
	saveAs("tiff", outputDir+"/Minimum_"+files[i]);
	run("Close All");
}

setBatchMode(false);
print("Finished analysis");
```

I ran it on some really busy images...
These are some max projections of the tested images:
Top row - raw DNA channel
Bottom row - calculated minimum image
![image](https://user-images.githubusercontent.com/45814595/96279271-bc35ea00-0fd6-11eb-8a23-170a20b1bd3b.png)

So I think it does a good job at cleaning up. Sometimes there is more than one bright object left, as in the sample on the right, but in that case I think the most obvious logic would be to take the most central object or exclude border objects.

**Comment by tischi** [comment]
Looks interesting!

I am playing now with an approach where I do an initial segmentation in _both channels_ and then, if there are multiple objects in the DNA channel, we could only keep the objects that overlap with the initial spindle segmentation... do you think that makes sense?

**Comment by tischi** [comment]
Would it be OK to remove all objects (both spindle and dna) in the initial segmentation which touch the image border in the x or y direction? I think that would also clean up quite some stuff!

**Comment by tischi** [comment]
> I am playing now with an approach where I do an initial segmentation in both channels and then, if there are multiple objects in the DNA channel, we could only keep the objects that overlap with the initial spindle segmentation... do you think that makes sense?

Naah, the issue there is that sometimes the metaphase plate is much dimmer than other bright DNA objects, so the metaphase plate is not detected at all.  So, one of your ideas above is probably better....

**Comment by tischi** [comment]
@TobiasKletter 

I think the best would be to try to roughly segment the spindle initially and then determine the initial DNA threshold within that mask. Could you play a bit more with the structure tensor idea for the spindle channel that I outlined above?

(Note: I played a bit with your good ideas of a minimum filter but it did not work robustly for me. I think one issue is that the spindle typically is quite dark where the metaphase plate is.)

Could you play again with the Hessian and Structure

**Comment by tischi** [comment]
What could be also interesting is to **_multiply_ the DNA image with the structure tensor filtered version of the tubulin image**.
Could you try this?

**Comment by TobiasKletter** [comment]
> What could be also interesting is to **_multiply_ the DNA image with the structure tensor filtered version of the tubulin image**.
> Could you try this?

Yep I'll have a go at this!

**Comment by tischi** [comment]
@TobiasKletter 
Above you also find a filter to enhance the DNA. You could also try applying this and then multiply:

```
RegionOfInterestImage = LargestHessian(DNA) * MiddleStructure(Tubulin)
```

**Comment by TobiasKletter** [comment]
Hi @tischi 

I've been playing around with different combinations of multiplications with filtered vs non-filtered channels, and this to me seems the most consistently clean operation:

![image](https://user-images.githubusercontent.com/45814595/102601521-91742900-4120-11eb-801a-a525be841882.png)

It's basically the following:
1. normalize channels as in macro above
2. create minimum image (minimum between the normalized channels)
3. Multiply minimum image with normalized tubulin channel

Using the filters as you suggested (Hessian for DNA, structure for tub) mostly worked well, but there are some annoyingly stubborn cases, so I've been trying to optimize on these and the operation above just worked best for all. 
(Here is one example of the difficult inputs: 
 https://www.dropbox.com/s/iffm060pvy3u6ss/HARD_20200806-164803_HighZoom--W0000--P0004-T0070--0002-1.tif?dl=0) 

I will test this now on a bigger image set and post an update!

> (Note: I played a bit with your good ideas of a minimum filter but it did not work robustly for me. I think one issue is that the spindle typically is quite dark where the metaphase plate is.)

So far I never had this problem, but I am going to explicitly look for some spindles where I see this and include them in the testing set.

**Comment by tischi** [comment]
Thanks!

Could you do me a favour and also post a macro here for the other approach `RegionOfInterestImage = LargestHessian(DNA) * MiddleStructure(Tubulin)`?

Then I could run the two macros side by side on some test data sets and try to understand what is going on...

**Comment by tischi** [comment]
...and as a starting point I would always use (250 nm)^3 isotropic images...

**Comment by TobiasKletter** [comment]
Macro 1 (FeatureJ...):

```
inPath = "/some/path";
outPath = "/some/other/path";

fileList = getFileList(inPath);

for (i = 0; i < fileList.length; i++) 
	{
	fileName = inPath + "/" +  fileList[i];
	print("Opening next: " + fileName);
	open(fileName);	

	DNA_channel = "C2-"+fileList[i];
	tub_channel = "C1-"+fileList[i];

	run("Split Channels");
	selectWindow(DNA_channel);
	run("FeatureJ Hessian", "largest absolute smoothing=3.0");
	rename("Hessian DNA");

	selectWindow(tub_channel);
	run("FeatureJ Structure", "middle smoothing=1.0 integration=3.0");
	rename("Structure Tubulin");
	
	imageCalculator("Multiply create stack", "Hessian DNA", "Structure Tubulin");
	saveAs("tif", outPath + "/" + "Filtered_Multiply_"+fileList[i]);
	
	run("Close All");
	}
print("Done.");

```

Macro 2 (Minimum operation):
```
inputDir = "/some/path";
outputDir = "/some/other/path";

fileList = getFileList(inputDir);

for (i = 0; i < fileList.length; i++) 
        {
	fileName = inputDir + "/" +  fileList[i];
	print("Opening next: " + fileName);
	open(fileName);	
 
	run("Duplicate...", "duplicate channels=1");
	rename("Channel_1");
	run("32-bit");
	run("Z Project...", "projection=[Median]");
	List.setMeasurements("limit");
	median_channel_1 = List.getValue("Median");
	stdev_channel_1 = List.getValue("StdDev");
	print("Median of channel 1= " + median_channel_1);
	print("StDev of channel 1 (TODO MAD?!)= " + stdev_channel_1);
	selectWindow("Channel_1");
	run("Subtract...", "value=median_channel_1 stack");
	run("Divide...", "value=stdev_channel_1 stack");
	

	selectWindow(fileList[i]);
	run("Duplicate...", "duplicate channels=2");
	rename("Channel_2");
	run("32-bit");
	run("Z Project...", "projection=[Median]");
	List.setMeasurements("limit");
	median_channel_2 = List.getValue("Median");
	stdev_channel_2 = List.getValue("StdDev");
	print("Median of channel 2= " + median_channel_2);
	print("StDev of channel 2 (TODO MAD?!)= " + stdev_channel_2);
	selectWindow("Channel_2");
	run("Subtract...", "value=median_channel_2 stack");
	run("Divide...", "value=stdev_channel_2 stack");
	

	imageCalculator("Min create stack", "Channel_1", "Channel_2");
	rename("MIN");
	imageCalculator("Multiply create stack", "MIN", "Channel_1");
	
	saveAs("tiff", outputDir+"/Minimum_MultiplyTub_"+fileList[i]);
	
	run("Close All");
        }
print("Done.");
```

**Comment by tischi** [comment]
> So far, this strategy seems to always leave considerable signal in the spindle center where tubulin and DNA overlap

I think the reason I am conceptually not sure about the "minimum" approach is that we normally see that the spindle is rather dim exactly at the metaphase plate, isn't it?

**Comment by tischi** [comment]
I have the ambition that we implement something that makes biological sense.
I think saying "in interphase cells tubulin and dna exclude each other however during mitosis they are in contact" makes sense, motivating the idea of the minimum filter!
But how do you motivate that you multiply in the end again with the tubulin channel?

**Comment by tischi** [comment]
I played some more but I have to admit your method works best so far 👍 

I would suggest:
1. Play with https://imagej.nih.gov/ij/docs/menus/process.html#enhance instead of the median/sdev based normalisation, does it improve? I would hope that some of it would help to suppress very bright DNA spots even more.
2. Think about a biological/mathematical motivation for your method, especially for multiplying again with the tubulin channel. Think like you would be giving a talk and have to explain your biophysical audience why you did what you did 😉

**Comment by tischi** [comment]
3. Regarding the fact that the tubulin in the spindle tends to be dimmer exactly where the DNA is I thought we could try a morphological closing:

```
// select tubulin

// reduce noise
run("Median 3D...", "x=1 y=1 z=1"); 

// fill gaps within the spindle using a closing(im) = max(min(im))
// the radius here is on purpose larger than for the noise reduction
run("Maximum 3D...", "x=2 y=2 z=2");
run("Minimum 3D...", "x=2 y=2 z=2");
``` 

### Raw data
![image](https://user-images.githubusercontent.com/2157566/102749633-a6d99500-4364-11eb-84fb-b265f1bafc04.png)

### After closing
![image](https://user-images.githubusercontent.com/2157566/102749645-ab05b280-4364-11eb-9d94-c1bd7c508f47.png)

**Comment by TobiasKletter** [comment]
> I have the ambition that we implement something that makes biological sense.
I think saying "in interphase cells tubulin and dna exclude each other however during mitosis they are in contact" makes sense, motivating the idea of the minimum filter!

Regarding the biological justification: We do expect some tubulin intensity overlapping with the DNA signal in the spindle midzone, mainly because of (non-kinetochore) microtubules of one pole extending into the opposite half. On top of that, at the default voxel size of 0.25 um the strong signal of the kinetochore microtubules probably adds to the signal overlap here, too.

> But how do you motivate that you multiply in the end again with the tubulin channel?

Good point... I think this was only a happy accident when I played with different things and seemingly gave the best results. It even might not be more effective than doing only the minimum filter operation 

> I would suggest:
Play with https://imagej.nih.gov/ij/docs/menus/process.html#enhance instead of the median/sdev based normalisation, does it improve? I would hope that some of it would help to suppress very bright DNA spots even more.

I will test this, thanks!
I will also check out the morphological closing operation.

**Comment by TobiasKletter** [comment]
Small update:

> Play with https://imagej.nih.gov/ij/docs/menus/process.html#enhance instead of the median/sdev based normalisation

Unfortunately, I cannot get it to work properly, I think. Seems like they removed some functions here? 
I came up with a work-around to loop through the slices and do this:
`run("Enhance Contrast...", "saturated=0.3 equalize")`
testing different percentages for the "saturated" argument, but this of course only looked at the histogram of the current slice, and basically did not help much. 
Is there a way to trim the histogram at a certain percentile by default?

All in all I still see the most consistently clean results with the Minimum calculation + multiplication with the tubulin channel.
Below is a selection of what I would judge to be the most tricky images, mainly because of bad mitotic DNA stainings and busy neighbourhood.

- Left is the raw DNA input, 
- Middle is the Minimum calculation, 
- Right is the Min calculation with tubulin multiplication afterwards
(both based on the script in https://github.com/tischi/spindle3d/issues/11#issuecomment-748171046 but with the denoising + closing step https://github.com/tischi/spindle3d/issues/11#issuecomment-748808774)

![image](https://user-images.githubusercontent.com/45814595/103652053-caa2fa80-4f62-11eb-8bbf-7198e32c5985.png)

Seems like the multiplication with the tubulin channel in the end nicely boosts the signal of the "good" overlaps between DNA and tubulin, even in images with extremely crappy mitotic DNA signal-to-noise, while it almost eradicates the annoying apoptotic DNA. This would also be my biological and technical motivation for going with this approach. 😄

**Comment by tischi** [comment]
@TobiasKletter 
I am trying to add an option to the advanced command that my code is running your macro internally.
I asked about this here: https://forum.image.sc/t/execute-imagej-macro-from-java/47278
...and it seems that should be possible.
Like this you would be free to explore and also adapt.
I will let you know whether I get it working!

**Comment by tischi** [comment]
I think I am getting somewhere....
What's missing in your code is a final step to create a region of interest, which I think should be a binary mask image.
I'll just add some auto-threshold algorithm to your macro but maybe you could explore what works best.

**Comment by TobiasKletter** [comment]
Yes I started working on something but didn't finish, the idea was:
- Minimum filter * tubulin channel
- strong Gaussian filter (sigma 5 -10)
- Auto-threshold (default algorithm) 
Since this might lead to multiple blobs: 
- connected component analysis (via MorphoLibJ), kill border objects, keep largest object


```
run("Gaussian Blur...", "sigma=10 stack");
setOption("BlackBackground", true);
run("Convert to Mask", "method=Default background=Default black");
run("Connected Components Labeling", "connectivity=6 type=[16 bits]");
run("Remove Border Labels", "left right top bottom front back");
run("Fill Holes (Binary/Gray)");
run("Keep Largest Label");
```

But since the normalized images create negative pixels, the auto-thresholding doesn't work always. Any ideas? I will try to fix this!

**Comment by tischi** [comment]
For me, this works on floating point images:

```
setAutoThreshold("Huang dark stack");
run("Convert to Mask", "method=Huang background=Dark black");
```

Could we have a convention that in your macro the `Tubulin` image is always channel 1?
In my code I can control this, because I have to create the input image for your macro.

**Comment by tischi** [comment]
Another question: We have no multiple DNA objects. How do we decide which one to keep? Maybe just the largest one?

**Comment by TobiasKletter** [comment]
OK the problem was the "remove border labels" step, because due to the heavy blurring, the object of interest might after thresholding end up touching the bottom or top of the stack. 
So this works for me quite nicely:

```
run("Gaussian Blur 3D...", "x=5 y=5 z=5");
setOption("BlackBackground", true);
setAutoThreshold("Triangle dark no-reset stack");
run("Convert to Mask", "method=Triangle background=Dark black");
run("Morphological Filters (3D)", "operation=Opening element=Cube x-radius=5 y-radius=5 z-radius=5");
run("Connected Components Labeling", "connectivity=6 type=[16 bits]");
run("Remove Border Labels", "left right top bottom"); // labels are still free to touch floor or roof of stack 
run("Fill Holes (Binary/Gray)");
run("Keep Largest Label");
```

> Could we have a convention that in your macro the Tubulin image is always channel 1?

Yes, sure! 👍 

> Another question: We have now multiple DNA objects. How do we decide which one to keep? Maybe just the largest one?

So for now I added this "Keep Largest Label" step exactly for this. Another idea could maybe be to select for the most central label?

**Comment by tischi** [comment]
> So for now I added this "Keep Largest Label" step exactly for this. Another idea could maybe be to select for the most central label?

Ok, I also changed my code to keep the largest label. Let's see how that work. The most central one was what I always wanted but you insisted that we cannot rely on this ;-)

Here is the new version, with the advanced command allowing to have a macro.

https://oc.embl.de/index.php/s/3fQq7j3sVjoUI2b

Short zoom? I send you a mail!

**Comment by tischi** [comment]
This is the macro that I used for testing:
```
setBatchMode(true); // hides all intermediate results
rename("input");

run("Duplicate...", "duplicate channels=1");
rename("Channel_1");
run("32-bit");
run("Z Project...", "projection=[Median]");
List.setMeasurements("limit");
median_channel_1 = List.getValue("Median");
stdev_channel_1 = List.getValue("StdDev");
print("Median of channel 1= " + median_channel_1);
print("StDev of channel 1 (TODO MAD?!)= " + stdev_channel_1);
selectWindow("Channel_1");
run("Subtract...", "value=median_channel_1 stack");
run("Divide...", "value=stdev_channel_1 stack");

selectWindow("input");
run("Duplicate...", "duplicate channels=2");
rename("Channel_2");
run("32-bit");
run("Z Project...", "projection=[Median]");
List.setMeasurements("limit");
median_channel_2 = List.getValue("Median");
stdev_channel_2 = List.getValue("StdDev");
print("Median of channel 2= " + median_channel_2);
print("StDev of channel 2 (TODO MAD?!)= " + stdev_channel_2);
selectWindow("Channel_2");
run("Subtract...", "value=median_channel_2 stack");
run("Divide...", "value=stdev_channel_2 stack");

imageCalculator("Min create stack", "Channel_1", "Channel_2");
rename("MIN");
imageCalculator("Multiply create stack", "MIN", "Channel_1");

setAutoThreshold("Huang dark stack");
run("Convert to Mask", "method=Huang background=Dark black");

setBatchMode("show"); // shows the result image (must be there)
```

**Comment by tischi** [comment]
@TobiasKletter 

Right now I am using the region of interest from your macro **_only_** in order to limit the region in which I determine the initial DNA threshold. While this helps not too choose the threshold too high, it does not prevent later that the actual metaphase DNA touches DNA from other cells, merges with them and thus touches the image border and therefore is rejected. You can see this if you look at the intermediate results. I added more diagnostic output images there. 

To avoid this we somehow would need another segmentation algorithm for the DNA, which does not only rely on a threshold.

However, maybe first see whether the new approach helps at least in the cases where the metaphase DNA is isolated from potentially bright DNA in the surrounding cells.

**Comment by TobiasKletter** [comment]
> Ok, I also changed my code to keep the largest label. Let's see how that work. The most central one was what I always wanted but you insisted that we cannot rely on this ;-)

OK true ;) but after the "cleanup" with the minimum filter etc., I think it's fair to assume the central object is the object of interest, at least that was my impression after doing the the 3D Gaussian.

> While this helps not too choose the threshold too high, it does not prevent later that the actual metaphase DNA touches DNA from other cells, merges with them and thus touches the image border and therefore is rejected. You can see this if you look at the intermediate results. I added more diagnostic output images there.

Could morphological erosion or opening help to disconnect the objects? Or is this already the volumetric mask and we don't want to tinker with that?

> Here is the new version, with the advanced command allowing to have a macro.

Thanks I'll test it today!

**Comment by TobiasKletter** [comment]
I played around with the most problematic samples and it mostly does not help much, sadly.

In many cases this appears:
```
Exception during computation: 
net.imglib2.exception.InvalidDimensionsException: Expected only positive dimensions but got: [0]
```

> To avoid this we somehow would need another segmentation algorithm for the DNA, which does not only rely on a threshold.

I am now thinking: could we be so drastic and actually crop around the mitotic DNA? The ROI macro does pretty reliably pinpoint the center or ROI of the object, can we exclude all DNA voxels in a radius of user-defined microns before thresholding? It's a bit brute force and not elegant, but I think something like this would work wonders for cells in organoids or tissues. 

> Here is the new version, with the advanced command allowing to have a macro.

I think it's a really cool idea to make the ROI detection interactive, but can we make this optional? Right now the advanced version requires a path...

**Comment by tischi** [comment]
I replaced the jar with a new version (same download link).

Could you please test:

1. The file should be optional now (not required)
2. It should throw a more informative error, namely: `"All DNA regions were touching the image border!"`

In addition, could you find one example where it does help? I would imagine that if there are additional bright DNA objects that are however spatially well separate from the metaphase plate it should help, right? I am thinking that for the publication that we are writing maybe we have to restrict ourselves to this scenario and discuss other cases, i.e. where the metaphase plate is connected by relatively bright intensities to the DNA in other cells as "future challenges in very crowded environments".
